packages/terra-functional-testing/src/commands/validates/screenshot.js

```diff
@@ -17,6 +17,7 @@ const screenshot = (testName, options = {}) => {
   const { selector } = options;
   const wrappedOptions = {
     name: testName,
+    updateScreenshots: global.Terra.serviceOptions.updateScreenshots,
     ...options,
```

Comment:
It might be worth spreading options first; otherwise it creates an unintended API.

```js
// Always update the screenshot.
Terra.validates.screenshot('name', { updateScreenshots: true }); // name too, but that's less concerning
```

Reply:
Good call. https://github.com/cerner/terra-toolkit/pull/568/commits/99e1b636372468bd4b470719b2c9af02185f6ba0